python/pyarrow/ipc.py

```diff
@@ -168,22 +170,26 @@ def serialize_pandas(df, nthreads=None, preserve_index=True):
     return sink.get_result()
 
 
-def deserialize_pandas(buf, nthreads=None):
+def deserialize_pandas(buf, nthreads=None, use_threads=False):
     """Deserialize a buffer protocol compatible object into a pandas DataFrame.
 
     Parameters
     ----------
     buf : buffer
         An object compatible with the buffer protocol
-    nthreads : int, defualt None
-        The number of threads to use to convert the buffer to a DataFrame,
-        default all CPUs
+    use_threads: boolean, default False
+        Whether to parallelize the conversion using multiple threads
 
     Returns
     -------
     df : pandas.DataFrame
     """
+    if nthreads is not None:
+        warnings.warn("`nthreads` argument is ignored, "
+                      "pass `use_threads` instead", FutureWarning,
+                      stacklevel=2)
```

Comment:
Same comment as above. Perhaps a helper function that issues the warning could be added to prevent the code duplication

Reply:
Right :-)